openshift-ci/deploy_ingress_node_firewall.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/bash
+
+ingress_node_firewall_dir="$(dirname $(readlink -f $0))"
+source ${ingress_node_firewall_dir}/common.sh
+
+INGRESS_NODE_FIREWALL_IMAGE_BASE=${INGRESS_NODE_FIREWALL_IMAGE_BASE:-$(echo "${OPENSHIFT_RELEASE_IMAGE}" | sed -e 's/release/stable/g' | sed -e 's/@.*$//g')}
+INGRESS_NODE_FIREWALL_IMAGE_TAG=${INGRESS_NODE_FIREWALL_IMAGE_TAG:-"ingress-node-firewall"}
+INGRESS_NODE_FIREWALL_DAEMON_IMAGE_TAG=${INGRESS_NODE_FIREWALL_DAEMON_IMAGE_TAG:-"ingress-node-firewall-daemon"}
+export NAMESPACE=${NAMESPACE:-"openshift-ingress-node-firewall"}
+
+rm -f ingress-node-firewall-operator-deploy/bundle.Dockerfile
+rm -rf ingress-node-firewall-operator-deploy/bundle
+
+cp ../bundle.Dockerfile ingress-node-firewall-operator-deploy 
+cp -r ../bundle/ ingress-node-firewall-operator-deploy/bundle 
```

Comment:
why are we doing this ?

Reply:
because we are building the index image in a container, and for the build we need the bundle and docker files (https://github.com/openshift/ingress-node-firewall/pull/138/files#diff-ada7b32c7c747e77d12f544afad91cbae16e9533234b866e10718fca9acb78c9R12). all is copied to the container and built here https://github.com/openshift/ingress-node-firewall/pull/138/files#diff-2da6abdb5a33ede3a2aefa3d1f31a4b48eb4d64f3aaf401254f4c57270aead21R74